modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,109 @@
+= Split Aggregate Scope
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule, split, aggregate, scope
+
+The Split Aggregate scope splits a collection and processes each part with the components within it in parallel (subject to any limitation in concurrency that has been configured).
```

Comment:
I would prefer to sin of extensive, that of short. Is up to you, but maybe is not too clear that each "processing" will transform each element. What do you thing?